### DISCUSSION TITLE
Continuos Delivery

### DISCUSSION THREAD

**Discussion by jordimassaguerpla** [Project-MONAI/monai-deploy-app-sdk#480] [Ideas]
Hi
I understand we can use monai-deploy-app-sdk to package the model into a container. But then, what? How do you envision we deploy that to "production" ? Which CD framework could be used for that task?

**Comment by MMelQin** [comment]
Good question. Like any other container workloads, there are many ways to host/run a containerized application.

- A MAP can be run as a standalone container processing a batch of files and save results. I have a guide on how this works on this board 
- MONAI Deploy with its Informatics Gateway, Workflow Manager, and TaskManager with docker plug-in, can integrate with external systems, e.g. PACS, to receive DICOM studies, trigger MAP as a task, and then export results to PACS in DICOM instances
- NYU's [Mecure](https://cai2r.net/resources/mercure/) supports MAP as a predefined processor
- FlyWheel supports MAP as a Gear
- Nuance, Answer Digital/NHS AIDE, also adopted MAP
- MAP can be used and hosted as an Nvidia Inference Microservice

An example list of adoptions and ways to host MAP are [here](https://github.com/Project-MONAI/monai-deploy-app-sdk/tree/main/platforms) in the repo.
